bench/algorithm/helloworld/1.rs

```diff
@@ -1,7 +1,3 @@
-fn main() {
-    let n = std::env::args_os()
-        .nth(1)
-        .and_then(|s| s.into_string().ok())
-        .unwrap_or(String::from(""));
-    println!("Hello world {}!", n)
+fn main() {    
+    println!("Hello world {}!", std::env::args().nth(1).unwrap_or("".to_owned()));
```

Comment:
`.unwrap_or("".to_owned())` this is still doing an unnecessary heap allocation, use `.unwrap_or_default()` or `.unwrap_or_else(|| "".to_owned())` instead